test/handlers/FlexVotingClientHandler.sol

```diff
@@ -0,0 +1,365 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+import {Test, console2} from "forge-std/Test.sol";
+import {IVotes} from "@openzeppelin/contracts/governance/utils/IVotes.sol";
+import {IGovernor} from "@openzeppelin/contracts/governance/Governor.sol";
+import {EnumerableSet} from "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
+
+import {GovernorCountingFractional as GCF} from "src/GovernorCountingFractional.sol";
+import {IVotingToken} from "src/interfaces/IVotingToken.sol";
+import {IFractionalGovernor} from "src/interfaces/IFractionalGovernor.sol";
+import {MockFlexVotingClient} from "test/MockFlexVotingClient.sol";
+import {GovToken} from "test/GovToken.sol";
+import {FractionalGovernor} from "test/FractionalGovernor.sol";
+import {ProposalReceiverMock} from "test/ProposalReceiverMock.sol";
+
+contract FlexVotingClientHandler is Test {
+  using EnumerableSet for EnumerableSet.AddressSet;
+  using EnumerableSet for EnumerableSet.UintSet;
+
+  MockFlexVotingClient flexClient;
+  GovToken token;
+  FractionalGovernor governor;
+  ProposalReceiverMock receiver;
+
+  uint128 public MAX_TOKENS = type(uint128).max;
+
+  // The number of actors that must exist before we allow a proposal to be
+  // created.
+  uint8 public PROPOSAL_THRESHOLD = 70;
+
+  EnumerableSet.UintSet internal proposals;
+  EnumerableSet.AddressSet internal voters;
+  EnumerableSet.AddressSet internal actors;
+  address internal currentActor;
+
+  // Maps proposalIds to sets of users that have expressed votes but not yet had
+  // them cast.
+  mapping(uint256 => EnumerableSet.AddressSet) internal pendingVotes;
```

Comment:
```suggestion
  mapping(uint256 => EnumerableSet.AddressSet) internal ghost_pendingVotes;
```

Reply:
I don't think this is really a ghost variable, since it's internal-only. Ghost variables track state so that it can be compared at test time. That's not what this does.